src/ngx_http_lua_socket_tcp.c

```diff
@@ -265,6 +266,9 @@ ngx_http_lua_inject_socket_tcp_api(ngx_log_t *log, lua_State *L)
     lua_pushcfunction(L, ngx_http_lua_socket_tcp_settimeouts);
     lua_setfield(L, -2, "settimeouts"); /* ngx socket mt */
 
+    lua_pushcfunction(L, ngx_http_lua_socket_tcp_getsockfd);
+    lua_setfield(L, -2, "getsockfd"); /* ngx socket mt */
```

Comment:
The `sock` word in the method name is redundant. It cannot really be something else. It can only be called on cosocket objects anyway. Or are you following LuaSocket's Lua API here? (I couldn't find such an API in LuaSocket).